pyxem/tests/test_utils/test_indexation_utils.py

```diff
@@ -20,7 +20,34 @@
 
 from pyxem.utils.indexation_utils import (crystal_from_template_matching,
                                           crystal_from_vector_matching,
-                                          match_vectors)
+                                          match_vectors,
+                                          normalized_correlation,
+                                          fast_correlation)
+
+
+def test_normalized_correlation():
+    nb_pixels = 3
```

Comment:
Would be happy for these to just read:
`assert np.allclose(normalized_correlation(3,1,1/3,[1,0,0],[1,0,0]),1)`

Not sure explicitly calculating helps very much and just adds dead line (also "mean" is preferred to "average")